myconverter/converter.py

```diff
@@ -0,0 +1,78 @@
+from PIL import Image
+import pyheif
+import fitz
+import shutil
+import os
+
+
+class MyConverter:
+    def __init__(self, input_file_path: str, output_file_path: str) -> None:
+        """
+        Initialize MyConverter class
+        Parameters:
+            input_file_path (str): input file path
+            output_file_path (str): output file path
+        """
+        self.input_file_path = input_file_path
+        self.output_file_path = output_file_path
+        self.ext = self.file_extension()
+
+    def file_extension(self) -> str:
```

Comment:
Code quality: Recommend using verb at the start of the name of the function which would return value.